client/modules/User/components/ResetPasswordForm.jsx

```diff
@@ -24,7 +24,7 @@ function ResetPasswordForm(props) {
       <Button
         type="submit"
         disabled={submitting || invalid || pristine || props.user.resetPasswordInitiate}
-      >Send Password Reset Email
+      >{props.t('ResetPasswordForm.Submit')}
```

Comment:
Is there a reason why this line uses `props.t(...)` but lines 14 and 17 use `t.(...)`?

Reply:
My bad, I forgot to update the last occurrence of props.t to t. I will update that